vars/microserviceBuilderPipeline.groovy

```diff
@@ -301,17 +316,22 @@ def getDeployBranch () {
   return deployBranch
 }
 
-def initalizeHelm (String tillerNamespace) {
-  container ('helm') {        
-    sh "helm init --skip-refresh"    
+def initalizeHelm (String tillerNamespace, String helmSecret) {
+  container ('helm') {
+    def options = ""
+    if (helmSecret) {
+       echo "initializing with --client only"
+       options += "--client-only "
```

Comment:
We should now always be` --client-only`. Jenkins should never be creating a Tiller.